cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     # v13 is mimic, v14 is nautilus, and v15 is octopus.
     # RECOMMENDATION: In production, use a specific version tag instead of the general v14 flag, which pulls the latest release and could result in different
```

Comment:
How about if we add an example of the specific tag with the date included and a comment about why they should consider using it?